### PR TITLE
Account switcher optimizations

### PR DESCRIPTION
**Pull request by AndyScherzinger**
cherry picking and adaption of oC user picker/list changes:
- bugfix for app being close when you navigate from drawer, to create user and then do a back press
- clicking a user item in the Manage accounts user list let's you switch the user this way
- Manage accounts user list items now show the user's name (like in the drawer) and also puts a user-badge on the avatar of the currently active user
- Translations cleanup

cc @tobiasKaminsky 

device-2016-10-11-160650

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -41,7 +41,6 @@ dependencies {
     compile "com.android.support:design:${supportLibraryVersion}"
     compile 'com.jakewharton:disklrucache:2.0.2'
     compile "com.android.support:appcompat-v7:${supportLibraryVersion}"
-    compile "com.android.support:cardview-v7:${supportLibraryVersion}"
```
Comment (tobiasKaminsky) [inline build.gradle]: Why is this removed? Don't we need it?

File: res/layout/toolbar_standard.xml [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or any later version.
 
   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
-  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-  GNU General Public License for more details.
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
```
Comment (tobiasKaminsky) [inline res/layout/toolbar_standard.xml]: double whitespace, also in some other files

File: src/com/owncloud/android/ui/activity/ManageAccountsActivity.java [hunk 3]
```diff
@@ -94,10 +104,17 @@ protected void onCreate(Bundle savedInstanceState) {
         setAccount(AccountUtils.getCurrentOwnCloudAccount(this));
         onAccountSet(false);
 
-        mAccountListAdapter = new AccountListAdapter(this, getAccountListItems());
+        mAccountListAdapter = new AccountListAdapter(this, getAccountListItems(),mTintedCheck);
```
Comment (tobiasKaminsky) [inline src/com/owncloud/android/ui/activity/ManageAccountsActivity.java]: add whitespace

**Comment by AndyScherzinger** [comment]
It is pulled into beta :D already!

File: build.gradle (hunk 1 above)
Comment (AndyScherzinger) [inline build.gradle]: nope, thus removed :smiley:

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky change has been in the wild via the beta release for a while, code review changes have been implemented. Imho ready to be merged now :rocket:

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky merge?

**Comment by AndyScherzinger** [comment]
Yes, removing the translations is on purpose since they aren't in use and also contained translations not present in the master strings.xml (which leads to unnecessary warnings during the build)
